tests/Microsoft.Identity.Test.Unit/BrokerTests/WamTests.cs

```diff
@@ -601,6 +601,55 @@ public async Task ATI_WithDefaultUser_Async()
             }
         }
 
+        [TestMethod]
+        public async Task ATI_WithDefaultUser_OrganizationsWorkaround_Async()
+        {
+            // Arrange
+            using (var harness = CreateTestHarness())
+            {
+                var requestParams = harness.CreateAuthenticationRequestParameters(TestConstants.AuthorityOrganizationsTenant);
+
+                var wamAccountProvider = new WebAccountProvider("id", "user@contoso.com", null);
+                var webTokenRequest = new WebTokenRequest(wamAccountProvider);
+
+                // will use the AAD provider because the authority is organizaitons
+                _webAccountProviderFactory
+                    .GetAccountProviderAsync(TestConstants.AuthorityHomeTenant)
+                    .ReturnsForAnyArgs(Task.FromResult(wamAccountProvider));
+
+                _aadPlugin.CreateWebTokenRequestAsync(
+                    wamAccountProvider,
+                    requestParams,
+                    isForceLoginPrompt: false,
+                    isAccountInWam: true,
+                    isInteractive: true)
+                    .Returns(Task.FromResult(webTokenRequest));
+
+                var webTokenResponseWrapper = Substitute.For<IWebTokenRequestResultWrapper>();
+                webTokenResponseWrapper.ResponseStatus.Returns(WebTokenRequestStatus.Success);
+                var webTokenResponse = new WebTokenResponse();
+                webTokenResponseWrapper.ResponseData.Returns(new List<WebTokenResponse>() { webTokenResponse });
+
+                _wamProxy.RequestTokenForWindowAsync(Arg.Any<IntPtr>(), webTokenRequest).
+                    Returns(Task.FromResult(webTokenResponseWrapper));
+                _aadPlugin.ParseSuccessfullWamResponse(webTokenResponse, out _).Returns(_msalTokenResponse);
```

Comment:
what does (out underscore) mean? is it just a variable?

Reply:
Good catch. It's called a Discard, a concept introduced in C# 7. https://docs.microsoft.com/en-us/dotnet/csharp/fundamentals/functional/discards

It's simple - if you don't care about a variable, you can use underscore and the compiler will not complain about things like "you assign a variable but you never use it".